ops/global/main.tf

```diff
@@ -48,3 +48,17 @@ module "insights" {
   rg_name       = data.azurerm_resource_group.rg.name
   tags          = local.management_tags
 }
+
+module "pd_demo" {
```

Comment:
Let's actually spell out Pagerduty here. Otherwise, `terraform state list` is pretty unreadable.

Reply:
Done and moved the state over.